docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       - ALLOWED_HOSTS=localhost,api
 
   dashboard:
-    image: ghcr.io/saleor/saleor-dashboard:3.16.1
+    image: ghcr.io/saleor/saleor-dashboard:3.17.1
```

Comment:
Could you put "latest" here - we have automation implemented, each latest release produces also tag "latest" on container

Reply:
Done